assets/js/pages/AnalyticsEula/AnalyticsEulaModal.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+
+import Modal from '@common/Modal';
+import Button from '@common/Button';
+import Input from '@common/Input';
+
+function AnalyticsEulaModal({
+  isOpen = false,
+  checked = false,
+  onChecked,
+  onEnable,
+  onCancel,
+}) {
+  return (
+    <Modal
+      className="!w-1/2"
+      title="Collection of Anonymous Metrics"
+      open={isOpen}
+      onClose={() => null}
+    >
+      Allow the collection of{' '}
+      <a
+        className="text-jungle-green-500 hover:opacity-75"
+        href="https://trento-project/docs"
+      >
+        anonymous metrics
+      </a>{' '}
+      to help improve Trento. This can also be managed also be managed in your
```

Comment:
typo:
`be managed` is duplicated

Reply:
I can't believe, I forgot about this one... 🤣